### DISCUSSION TITLE
v0.14.b3 (beta) tar.gz installs latest v0.14.3 release version instead

### DISCUSSION THREAD

**Discussion by aaronreek** [Louisvdw/dbus-serialbattery#434] [General]
I've been trying to install the beta release v0.14.b3 to try out the updated config/utils implementation, but the stable release v0.14.3 is always installed instead.
I've tried using wget and also saving the tar.gz on a usb stick and inserting in the Pi, but both methods give the same result (stable release v0.14.3 is installed, not the beta version).
I've also tried deleting my previous install before attempting to install the beta version, and that didn't work either.
Am I doing something wrong? Is there another way to install the beta version?

I have v0.14.3 stable release working no problem, just can't seem to update to the new beta version.

**Comment by mr-manuel** [comment]
~~~bash
cd /tmp

wget https://github.com/Louisvdw/dbus-serialbattery/archive/refs/heads/master.zip
unzip master.zip

cp /tmp/dbus-serialbattery-master/etc/dbus-serialbattery/ /data/etc

chmod +x /data/etc/dbus-serialbattery/*.sh
chmod +x /data/etc/dbus-serialbattery/*.py
chmod +x /data/etc/dbus-serialbattery/service/run
chmod +x /data/etc/dbus-serialbattery/service/log/run

bash /data/etc/dbus-serialbattery/installlocal.sh
~~~

**Reply by mr-manuel** [reply]
You can also try the `jkbms_ble` version even, if you are not using bluetooth. The branch is more up to date at the moment. https://github.com/Louisvdw/dbus-serialbattery/discussions/485
